medusa/scene_numbering.py

```diff
@@ -252,7 +252,8 @@ def get_scene_numbering(series_obj, scene_episode, scene_season=None):
     return new_sea, new_ep
 
 
-def get_scene_abs_number(series_obj, episode, season=None):
+def get_abs_number_from_scene_abs(series_obj, episode, season=None):
```

Comment:
Maybe `get_abs_number_from_scene` (more generic) since it works on both absolute and regular numbering? Otherwise I'd think it's for absolute numbering only.